tools/replicate_server_settings.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+require File.expand_path("../config/environment", __dir__)
+require 'trollop'
+
+opts = Trollop.options(ARGV) do
+  banner "USAGE:   #{__FILE__} -s <server id> -p <settings path separated by a /> \n" \
```

Comment:
I feel like something like `path/to/the/settings` might be more clear? Especially because `/>` looks like some kind of closing tag, so it might get ignored.

Reply:
done!